.asciidoctor/asciidoctor.css

```diff
@@ -1352,6 +1352,20 @@ div.unbreakable{
 .line-through{
     text-decoration:line-through
 }
+.columnA,
+.columnC {
```

Comment:
columnC isn't used, can we define only columnA?

Reply:
Yes, we can, I'll update the PR.